packages/react-three-rapier/src/components/Physics.tsx

```diff
@@ -293,7 +298,22 @@ export interface PhysicsProps {
   predictionDistance?: number;
 
   /**
-   * The Error Reduction Parameter in between 0 and 1, is the proportion of the positional error to be corrected at each time step
+   * Minimum number of dynamic bodies in each active island
+   *
+   * @defaultValue 128
+   */
+  minIslandSize?: number
+
+  /**
+   * Maximum number of substeps performed by the solver
+   *
+   * @defaultValue 1
+   */
+  maxCcdSubsteps?: number
+
+  /**
+   * The Error Reduction Parameter in between 0 and 1, is the proportion of the positional error to be corrected at each time step.
+   *
    * @defaultValue 0.8
```

Comment:
Inside rapier.js, the default value is 0.2 - yet we have 0.8 here. Is there a reason for this? If not, we should leave it at the value they suggest.

Directly from rapier.js at /rapier3d/dynamics/integration_parameters.ts
```
    /**
     * The Error Reduction Parameter in `[0, 1]` is the proportion of
     * the positional error to be corrected at each time step (default: `0.2`).
     */
    get erp(): number {
        return this.raw.erp;
    }
```


```suggestion
   * @defaultValue 0.8
```